tests/Unit/Services/VoucherEvaluator/VoucherEvaluatorTest.php

```diff
@@ -90,34 +154,65 @@ public function itCreditsWhenAFamilyIsPregnant()
     {
-        // Make extended evaluator
-        $evaluator = EvaluatorFactory::make("extended_age");
+        // get rules mods
+        $rulesMods = collect($this->rulesMods["credit-extended"]);
+
+        // Make evaluator
+        $evaluator = EvaluatorFactory::make($rulesMods);
 
         // Make our family
         $family = factory(Family::class)->create();
 
-        // Make a set of ineligible kids (no under school age)
-        $overSchool = factory(Child::class, 'overSchoolAge')->make();
-        $overSecondarySchool = factory(Child::class, 'overSecondarySchoolAge')->make();
+        // Make kids
+        $isPrimarySchool = factory(Child::class, 'isPrimarySchoolAge')->make();
+        $isOverPrimarySchool = factory(Child::class, 'isSecondarySchoolAge')->make();
 
         // Add the kids and check they saved
-        $family->children()->saveMany([$overSchool ,$overSecondarySchool]);
+        $family->children()->saveMany([$isPrimarySchool ,$isOverPrimarySchool]);
+        $this->assertEquals(2, $family->children()->count());
+
+        $evaluation = $evaluator->evaluate($family);
+
+        // Check it can find eligible children (0 vouchers)
+        // - because no under primary school-ers validate the primary school-ers.
+        $this->assertTrue($evaluation->getEligibility());
+        $this->assertEquals('3', $evaluation->getEntitlement());
+    }
+
+    /** @test */
+    public function itCreditsQualifiedPrimarySchoolChildrenButNotUnqualifiedOnes()
+    {
+        // get rules mods
+        $rulesMods = collect($this->rulesMods["credit-extended-qualified"]);
+
+        // Make evaluator
+        $evaluator = EvaluatorFactory::make($rulesMods);
+
+        // Make our family
+        $family = factory(Family::class)->create();
+
+        // Make a set of ineligible kids (no under primary school age)
+        $isPrimarySchool = factory(Child::class, 'isPrimarySchoolAge')->make();
+        $isOverPrimarySchool = factory(Child::class, 'isSecondarySchoolAge')->make();
+
+        // Add the kids and check they saved
+        $family->children()->saveMany([$isPrimarySchool ,$isOverPrimarySchool]);
         $this->assertEquals(2, $family->children()->count());
 
         // Run the evaluation
         $evaluation = $evaluator->evaluate($family);
         // Check it can't find any eligible children (0 vouchers)
-        // - because no under primary school-ers validate the under secondary school-ers.
+        // - because no under primary school-ers validate the primary school-ers.
         $this->assertFalse($evaluation->getEligibility());
         $this->assertEquals('0', $evaluation->getEntitlement());
 
-        // Add a kid that will make the child under secondary school age.
-        $underSchool = factory(Child::class, 'underSchoolAge')->make();
+        // Add a kid that will make the child at primary school age qualified.
+        $underPrimarySchool = factory(Child::class, 'betweenOneAndPrimarySchoolAge')->make();
```

Comment:
Is this the right test for extended family rule?
Should it be that having a kid at primary school, under one, or between one and primary will make the at secondary school children qualify?

Reply:
No, secondary schoolers don't get vouchers yet. 

normally
- pregnancy - 3
- under one - 6
- toddler - 3
- primary - 0

the credit-primary changes that to
- primary - 3

the credit-primary-qualified qualfies that with
- primary (with a toddler, under one,  or pregnancy) - 3